### PR TITLE
Implement Exercise "Secret Handshake"

### LINKED ISSUE #212: Implement Exercise "Secret Handshake"
Source: https://github.com/exercism/problem-specifications/tree/main/exercises/secret-handshake
Implementation in fork: https://github.com/bau-mann/abap/tree/secret_handshake_main
Related Issue: #211 
Please assign this issue to me.

### PR DESCRIPTION
**Pull request by bau-mann**
- resolves #212
- depends on #211

### CODE REVIEW

File: exercises/practice/secret-handshake/.meta/zcl_secret_handshake.clas.abap [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+CLASS zcl_secret_handshake IMPLEMENTATION.
+
+  METHOD get_commands.
+
+    DATA binary_zero TYPE x VALUE 0.
+    DATA(binary_code) = CONV xstring( code ).
+
+    DATA(codes_transl) = VALUE string_table(  ( `wink` ) ( `double blink` )  ( `close your eyes` )  ( `jump` ) ).
```
Comment (larshp) [inline exercises/practice/secret-handshake/.meta/zcl_secret_handshake.clas.abap:23]:
```suggestion
    DATA(codes_transl) = VALUE string_table( ( `wink` ) ( `double blink` )  ( `close your eyes` )  ( `jump` ) ).
```

**Comment by larshp** [comment]
cool, looks good to me, you can set the PR to "ready for review" 🎉 

image

**Comment by bau-mann** [comment]
Please note that I don't have access to a system with working abapGit at the moment. I have created and double-checked files package.devc.xml and zcl_secret_handshake.clas.xml by hand, but I cannot actually test whether abapGit is able to deserialize them properly.

**Comment by larshp** [comment]
from my point of view: as long as the exercism stuff works, then abapGit is nice to have, can always do a follow up PR to fix it

tho, not too many follow up PRs as it triggers updates to the students

**Comment by mbtools** [comment]
Almost. Just add a final newline to the class and testclass. Then it will deserialize without diffs:

image

image

**Review by mbtools — APPROVED** [review]
👍

**Comment by mbtools** [comment]
good to merge @larshp

**Comment by larshp** [comment]
merged, thanks @bau-mann

**Comment by mbtools** [comment]
Welcome to Exercism as a contributor 🎉

**Comment by bau-mann** [comment]
Really great, I was able to find and solve the exercise on exercism.org. Thank you for your support @larshp and @mbtools.

**Comment by mbtools** [comment]
Nice exercise! Unfortunately, my solution using some fancy ABAP features doesn't work with the transpiler (yet):

https://github.com/abaplint/transpiler/issues/901

**Pull request merged**